### ISSUE TITLE
Sortable table accessibility fix

### ISSUE THREAD

**Issue by chrispymm** [ministryofjustice/moj-frontend#1364]
## What

The arrows in the sortable table are read out by screenreaders, which provides a bad ux for screenreader users

## Why

We need to fix this as it is a WCAG fail

## Who needs to be involved

- Developer
- Accessibility Specialist

## Done

The arrows will no longer be read out by screenreaders, and the component will meet WCAG criteria
